src/a-runtime-console/kubernetes/ui/pipeline/build-stage-view/build-stage-view.component.html

```diff
@@ -16,11 +16,28 @@
       </div>
     </div>
     <div class="pipeline-container">
-      <div class="pipeline" *ngIf="!build.pipelineStages.length">
-        <div class="pipeline-stage no-stages">
-          <div id="pipeine-stage-name-no-stages" class="pipeline-stage-name">No stages have started.</div>
+      <div class="pipeline" *ngIf="!build.logURL && !build.pipelineStages.length">
```

Comment:
Maybe you could use something like this instead. 
`*ngIf="!build.logURL && !build.pipelineStages.length; else pipeline-started"`

Reply:
@joshuawilson There are three `*ngIf` that will require nested `*ngIf ... else`, will that be good?

Reply:
You can use it for just these 2 and leave the 3rd, which is different, as a separate `ngif`. It is only meant to clean up the code to make it easier to read. It doesn't effect functionality.

Reply:
Thanks Joshua , Have updated the the code using `<ng-container *ngIf=" ...` it'll help to keep it clean.  Let me know if this is fine. These 3 *ngIf's are like `if - else if - else`so leaving 3rd one won't help.